app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java

```diff
@@ -78,14 +79,15 @@
 
     private EditImageActivity mContext;
     public Bitmap mainBitmap;
+    private Bitmap originalBitmap;
     public ImageViewTouch mainImage;
-    private View cancel,save;
+    private View cancel,save,bef_aft;
```

Comment:
@vinaysajjanapu not mandatory. It is pointing because the variable `bef_aft` only used in the scope of `onCreate` block. 
So can be replaced by local variable